src/Symfony/Component/Console/Tests/Helper/HelperTest.php

```diff
@@ -33,9 +33,12 @@ public static function formatTimeProvider()
             [121,    '2 mins, 1 sec', 2],
             [3599,   '59 mins, 59 secs', 2],
             [3600,   '1 hr', 2],
+            [5199,   '1 hr, 26 mins, 39 secs', 1.4],
```

Comment:
The 3rd parameter must be an `int` representing the `$precision`, i.e. the number of units to be displayed).